test/units/plugins/connection/test_ssh.py

```diff
@@ -549,8 +549,8 @@ def test_incorrect_password(self, monkeypatch):
         self.conn.get_option = MagicMock()
         self.conn.get_option.return_value = True
 
-        exception_info = pytest.raises(AnsibleAuthenticationFailure, self.conn.exec_command, 'sshpass', 'some data')
-        assert exception_info.value.message == ('Invalid/incorrect username/password. Skipping remaining 5 retries to prevent account lockout: '
+        exception_info = pytest.raises(AnsibleConnectionFailure, self.conn.exec_command, 'sshpass', 'some data')
+        assert exception_info.value.message == ('sshpass error: Invalid/incorrect password. '
                                                 'Permission denied, please try again.')
```

Comment:
While you're here, mind improving the style a bit?
```suggestion
        expected_error = (
            r'^sshpass error: Invalid/incorrect password\. '
            r'Permission denied, please try again\.$'
        )
        with pytest.raises(AnsibleConnectionFailure, match=expected_error) as exception_info:
            self.conn.exec_command('sshpass', 'some data')
```